config/jest/environment/jsdom.mjs

```diff
@@ -13,6 +15,10 @@ class BrowserVirtualEnvironment extends TestEnvironment {
 
     // Ensure test fails for browser exceptions
     virtualConsole.on('jsdomError', (error) => process.emit('error', error))
+
+    // Add shared test globals
+    // componentsData, componentsDirectory, examplesDirectory
+    this.global.namespace ??= await globals()
```

Comment:
**nitpick**: That name confused me a bit, I didn't get what it was the namespace of. As it's effectively the cache we're reading from, could we name it just `cache` or something cache related?

Reply:
Yeah, it was a general "dumping ground" for shared things name! I'm happy with cache

```suggestion
    this.global.cache ??= await globals()
```